pkg/ui/workspaces/db-console/src/views/shared/components/summaryBar/index.tsx

```diff
@@ -109,11 +109,17 @@ export function SummaryValue(
 export function SummaryStat(
   props: SummaryStatProps & { children?: React.ReactNode },
 ) {
+  let classModifier = "number";
+
+  if (props.title === "Unavailable ranges" && props.value !== 0) {
+    classModifier = "alert-unavailable-ranges";
```

Comment:
I'd prefer if we didn't have a special case hidden inside `SummaryStat`. This creates coupling between the mostly generic component and the specific use case. Imagine if we change the text of "unavailable ranges", then we'd have to remember to modify it here too. Ideally, the use case can communicate its intent in a more generic way.

Here are some options you can pick between:
* Create a `SummaryStatAlert` component with the special case.
* `SummaryStat` can take an additional prop that configures it to show the number in red. This could be a boolean named `alert` which matches Alex's CSS suggestion. When it's true, you can set the different class modifier.

Reply:
Hi @dhartunian Thanks for suggestion,
I have updated the code.